docs/source/techniques/ISISPowder-Tutorials.rst

```diff
@@ -45,13 +45,13 @@ Copying instrument example files
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 Open your Mantid install location, by default this
 will be `C:\\MantidInstall` on Windows and `/opt/Mantid` on Linux.
-Open *scripts*, *diffraction*, *isis_powder*. 
+Open *scripts* - *Diffraction* - *isis_powder*. 
```

Comment:
nit: I'm assuming this is a path through directories? 
Might be more obvious with a notation such as `path>to>script>of>interest`